config/_examples/_cats_ioi/settings.json

```diff
@@ -1,9 +1,9 @@
 {
   "url":"https://imcs.dvfu.ru/cats",
-  "cid":"5974394",
+  "cid":"6849202",
   "type":"cats",
   "timeZone":"UTC+10",
-  "resultType":"IOI",
+  "resultType":"ICPC",
```

Comment:
This doesn't seem right

Reply:
Why?

Reply:
I change old id of contest to new. New contest - has a icpc format

Reply:
But the directory is named _cats_ioi. Let's create new _cats_icpc instead?